mcpgrafana.go

```diff
@@ -56,6 +58,17 @@ type GrafanaConfig struct {
 	// Debug enables debug mode for the Grafana client.
 	Debug bool
 
+	// EnableTracing enables OpenTelemetry instrumentation for both tool execution
+	// and HTTP requests, providing complete observability with perfect correlation
+	// between tool spans and HTTP request spans. Defaults to false for minimal overhead.
+	EnableTracing bool
+
+	// LogTraceArguments enables logging of tool arguments in OpenTelemetry spans
+	// when tracing is enabled. This should only be enabled in non-production 
+	// environments or when you're certain the arguments don't contain PII. 
+	// Defaults to false for safety. Requires EnableTracing to be true.
+	LogTraceArguments bool
```

Comment:
is this really helpful somewhere? can we debug log arguments passed to the tools before calling mcp instead?

Reply:
Is 'Log' the correct verb here? At first I thought this referred to regular logging, not including the args in span attributes, so it's a bit confusing.